docker-compose.yml

```
@@ -1,11 +1,102 @@
+      PROXY_ADDRESS_FORWARDING: "true"
+    volumes:
+      - ./keycloak-import:/opt/keycloak/data/import
+    ports:
+      - 8080:8080
+    depends_on:
+      - postgres
+    networks:
+      - ubi_nw
+
+  ubi-beneficiary-backend:
+    image: ubi-beneficiary-backend:latest
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 5 but found 4

(indentation)